docs/from_the_sources.md

```diff
@@ -23,11 +23,13 @@ $ node -v
 v8.9.1
 ```
 
+See [this page](https://nodejs.org/en/download/package-manager/) for instructions on how to install Node.
```

Comment:
Install via package manager will probably not install LTS version of node (older or too recent). What's your opinion @julien-f , should we add this or not?

Reply:
I think we can add this, it's clearly indicated that XO requires LTS.